src/model_signing/verifying.py

```diff
@@ -102,19 +90,14 @@ def set_hashing_config(self, hashing_config: hashing.Config) -> Self:
         self._hashing_config = hashing_config
         return self
 
-    def set_sigstore_dsse_verifier(
-        self,
-        *,
-        identity: str,
-        oidc_issuer: Optional[str] = None,
-        use_staging: bool = False,
+    def use_sigstore_verifier(
+        self, *, identity: str, oidc_issuer: str, use_staging: bool = False
```

Comment:
issuer no longer optional seems intentional?

Reply:
It is, it was a mistake to not have it optional at the start.

Reply:
From a design standpoint? The test in api_test.py worked for example

Reply:
Yes, but that's strange. All sigstore verification requires it. We can make it optional later, but making it required when it's optional breaks compatibility.

Reply:
Seems to imply optional here.
https://github.com/sigstore/sigstore-python/blob/310371e4fae59555db8007f01e5af3984bac7e57/sigstore/verify/policy.py#L433-L443

Reply:
Ah, I was looking at https://sigstore.github.io/sigstore-python/verify/#verifying-against-a-bundle where it was mandatory. Probably it got made optional more recently.